incubator/mattermost-team-edition/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "mattermost-team-edition.fullname" . }}
+  labels:
+    chart: {{ template "mattermost-team-edition.chart" . }}
+    app: {{ template "mattermost-team-edition.fullname" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "mattermost-team-edition.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/config.tpl") . | sha256sum }}
+      labels:
+        app: {{ template "mattermost-team-edition.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      {{- if not .Values.externalDB.enabled }}
+      initContainers:
+        - name: "init-mysql"
+          image: "appropriate/curl:latest"
+          imagePullPolicy: "IfNotPresent"
+          command: ["sh", "-c", "until curl --max-time 5 http://{{ .Release.Name }}-mysql:3306; do echo waiting for {{ .Release.Name }}-mysql; sleep 5; done;"]
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+          livenessProbe:
+            initialDelaySeconds: 90
+            timeoutSeconds: 5
+            periodSeconds: 15
+            httpGet:
+              path: /api/v4/system/ping
+              port: {{ .Values.service.internalPort }}
+          readinessProbe:
+            initialDelaySeconds: 15
+            timeoutSeconds: 5
+            periodSeconds: 15
+            httpGet:
+              path: /api/v4/system/ping
+              port: {{ .Values.service.internalPort }}
+          volumeMounts:
+          - mountPath: /mattermost/config/config.json
+            name: config-json
+            subPath: config.json
+          - mountPath: /mattermost/data
+            name: mattermost-data
+{{- with .Values.extraVolumeMounts }}
+{{ toYaml . | indent 12 }}
+{{- end }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+{{- with .Values.extraContainers }}
+{{ tpl . $ | indent 8 }}
+{{- end }}
+      volumes:
+        - name: config-json
+          configMap:
+            name: {{template "mattermost-team-edition.fullname" .}}-config-json
```

Comment:
Please be consistent. Add a space after `{{` am  before `}}`. Apply throughout the chart.

Reply:
done